### PR TITLE
[vtk] patch GCC13 compilation errors

### PR DESCRIPTION
**Pull request by Meakk**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- ~~[ ] SHA512s are updated for each updated download~~
- ~~[ ] The "supports" clause reflects platforms that may be fixed by this new version~~
- ~~[ ] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~~
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by Meakk** [comment]
@microsoft-github-policy-service agree

File: ports/vtk/missing-include-fixes.patch [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+--- a/ThirdParty/libproj/vtklibproj/src/proj_json_streaming_writer.hpp
++++ b/ThirdParty/libproj/vtklibproj/src/proj_json_streaming_writer.hpp
+@@ -31,6 +31,7 @@
+ 
+ /*! @cond Doxygen_Suppress */
+ 
++#include <cstdint>
+ #include <vector>
+ #include <string>
+
```
Comment (Adela0814) [inline ports/vtk/missing-include-fixes.patch:40]: Thanks for your PR. Please submit this change to the upstream, we need to wait for upstream approval first.

**Review by Adela0814 — DISMISSED** [review]

**Comment by Adela0814** [comment]
Note: I will be converting your PR to draft status. When you respond, please revert to "ready for review". That way, I can be aware that you've responded since you can't modify the tags.

File: ports/vtk/missing-include-fixes.patch (hunk 1 above)
Comment (dg0yt) [reply ports/vtk/missing-include-fixes.patch:40]: Adding the standard include, similar to what many other ports need, shouldn't require approval IMO,

Comment (Meakk) [reply ports/vtk/missing-include-fixes.patch:40]: This has already fixed upstream

**Review by dan-shaw — APPROVED** [review]

**Pull request merged**